fiftyone/server/db/mongo.py

```
@@ -0,0 +1,174 @@
+            foa.CountValues(
+                path,
+                _first=first,
+                _asc=asc,
+                _sort_by=sort_by,
+                _search=search,
+                _selected=selected,
+            ),
+        )
+        return ValuePickerResult(total=total, page=list(page))
+
+    async def lightning(self, dataset, *, input):
```

Ruff (0.15.12)

[error] Function argument `input` is shadowing a Python builtin

(A002)